dashboard.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
what is this file used for?

Reply:
Nothing yet! It’s in the template but unused at the moment. Forgot I copied that in there... do plan on potentially using it or something like it in the future.